### DISCUSSION TITLE
Raspberry Pi - Nicht erreichbar

### DISCUSSION THREAD

**Discussion by Exo84** [camueller/SmartApplianceEnabler#374] [Installation und Betrieb]
Hallo,

ich habe mich dran gemacht, den SAE auf meiner PI3 B zu installieren.
Habe es bis dato zweimal wiederholt, nun geht es dennoch nicht.

Ich halte mich an die Beschreibung.
Kurzfassung (in der jeweils Klammer evtl. Fehlerquellen, welche es sein könnten):

**SD-Karte beschreiben**
**Drei Files auf dem USB Stick speichern**
(Wenn ich im installer guid auf die Filename klicke geht nur ein neuer Tab mit dem Inhalt auf. Hier habe ich denn per Rechtsklick Speicher unter gemacht und direkt auf den Stick gespeichert install.config ist eine TXT Datei und die beiden anderen sind SH Dateien)
**SD-Karte rein**
**Strom an**
(Netzwerkkabel steckt auch drin)
**Warten bis die PI hochgefahren ist**
**USB Stick rein**
**Lichter gehen aus**
(geht)
**USB Stick raus**
**Lichter gehen an**
**Warten**
(Auch nach x Stunden bleibt die rote LED immer an. Lt. Installer guid sollte sie für eine Stunde ausgehen.)

Status: Die PI3 erscheint in der Fritzbox. http://IP-Adresse:8080 und auch :1000 in Chrome eingegeben:
Meldung immer ERR_CONNECTION_REFUSED

Leider kenne ich mich nicht mit Raspberry Pi aus und habe gehofft, dass es ohne Komplikationen läuft.
Lt. Installer guid sollte es ja Logs geben welche helfen könnten, aber ich komme ja nicht auf die PI drauf.

Könnt Ihr mir bitte noch einen Tipp geben?

Danke

**Comment by camueller** [comment]
> Lt. Installer guid sollte es ja Logs geben welche helfen könnten, aber ich komme ja nicht auf die PI drauf.

Warum kommst Du nicht auf den Raspberry Pi drauf? Wenn der Pi in der Fitzbox erscheint, läuft er ja, d.h.Du Dich via SSH mit ihm verbinden. Und dann kannst Du auch die [Installlation via Script](https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/InstallationViaScript_DE.md) durchführen.

**Comment by Exo84** [comment]
Es ging! Er ist via Browser erreichbar. Danke!

Kurze Zusammenfassung für diejenigen welche sich mit der PI usw. genau so wenig auskenne wie ich:

1. Schreiben des Raspberry Pi OS Images auf die SD-Karte
2. Windows Eingabeaufforderung starten
3. "ssh pi@Eure PI IP Adresse" --> eingeben
4. Passwort eingeben --> Google nach Standard Passwort fragen
5. "sudo bash" --> eingeben
6. "curl -sSL https://raw.githubusercontent.com/camueller/SmartApplianceEnabler/master/install/setup.sh | sh" --> eingeben
7. Warten (Rote LED war bei mir immer an?)
8. Browser öffnen und "http://PI IP Adresse:8080" --> Eingeben

**Reply by Exo84** [reply]
Kann mir nochmal jemand bitte helfen. 

Wie verbinde ich die PI jetzt mit meinem wlan?

**Comment by solanjo** [comment]
Hallo Exo84,

über die SSH-Verbindung gibst Du "sudo raspi-config" ein, um in die Konfiguration des PIs zu gelangen. Dort kannst Du unter "System Options" die relevanten Verbindungsdaten eingeben. Evtl. ist nötig, vorher noch die "Localisation Options" einzurichten.

Viel Spaß!

**Reply by solanjo** [reply]
Mit "sudo" bist Du Superuser! Als pi oder sae kannst Du die Config garnicht starten.

**Reply by Exo84** [reply]
hat geklappt danke!!!!
